src/main/kotlin/net/maizegenetics/phgv2/cli/Gvcf2Hvcf.kt

```diff
@@ -53,6 +54,16 @@ class Gvcf2Hvcf: CliktCommand(help = "Create  h.vcf files from existing PHG crea
     val dbPath by option(help = "Folder name where TileDB datasets and AGC record is stored.  If not provided, the current working directory is used")
         .default("")
 
+    // Default to using minHaplotypeSize over minHaplotypePercent.  If minHaplotypeSize is not set, then use minHaplotypePercent
+    // Our default minHapltoypePercent will be 5, but can be reset for testing.
+    val minHaplotypeSize by option(help= "Minimum size of haplotype to be included in hvcf")
```

Comment:
Similar to before, I thought we weren't messing with Haplotype size, just refRange size.